Remove all old technical documentation from gen3.org (#322)
uc-cdis/gen3.org: content/splash/index.md

---
identifier: content-deprecated

> title: "Notice: Content Deprecated"

Reviewer: ```suggestion
title: "Notice: Content Updated and Moved"
```

linktitle: content-deprecated
date: 2026-03-25
aliases:
  - /get-started/
  - /resources/
  - /resources/gen3-features
  - /resources/user
  - /resources/developer
  - /resources/operator
  - /resources/operator/index.html
  - /resources/faq
  - /resources/user/submit-data
  - /resources/user/access-data
  - /resources/developer/build-app
g3Teaser:

>   title: Notice - Content Deprecated

Reviewer: ```suggestion
  title: Notice - Content Updated and Moved
```

---

<section class="g3-bg__white">
  <div class="g3-outer-wrapper g3-flex-content">
    <div class="g3-space__padding-lg-top g3-space__padding-lg-bottom g3-col__65">
      <div class="g3-space__wrapper-gap-left">
        <h1 class="g3-space__margin-sm-bottom">
          {{< param "g3Teaser.title" >}}
        </h1>
        <p class="g3-space__margin-sm-bottom introduction">

>           The page you are looking for is no longer available. All Gen3 technical documentation has moved to our new site at <a href="https://docs.gen3.org" target="_blank">docs.gen3.org</a>. Please review the new documentation there.

Reviewer: ```suggestion
          We have new Gen3 documentation! All Gen3 technical documentation has moved to our new site at <a href="https://docs.gen3.org" target="_blank">docs.gen3.org</a>. The content you are looking for is likely there. Please review the new documentation, and reach out to us at support@gen3.org if you have any trouble.
```
